.github/workflows/update-version.yml

```diff
@@ -0,0 +1,24 @@
+name: Check TimeZone Version
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.102
```

Comment:
this will probably break once we bump the version in global.json, we can instead use the `build.sh` script so we download `dotnet` during the build:

```bash
./build.sh --projects $(pwd)/src/System.Runtime.TimeZoneData /t:UpdateToLatestVersion
```

Reply:
Hmm looks like this doesn't work since it doesn't recognize the `/t:UpdateToLatestVersion` target, @ViktorHofer any suggestions?

Reply:
yeah, that is much better.

Reply:
@ViktorHofer it doesn't work since it's trying to find the target in Build.proj from arcade:

```
/Users/alexander/.nuget/packages/microsoft.dotnet.arcade.sdk/5.0.0-beta.20478.3/tools/Build.proj : error MSB4057: The target "UpdateToLatestVersion" does not exist in the project.
```

Reply:
The --projects switch was probably not designed to be used that way. I suggest to bring these files from runtime along in this PR and use them instead:

- https://github.com/dotnet/runtime/blob/main/dotnet.cmd
- https://github.com/dotnet/runtime/blob/main/dotnet.sh

`./dotnet.sh build src/System.Runtime.TimeZoneData /t:UpdateToLatestVersion`